numba/parfors/parfor.py

```diff
@@ -1350,14 +1350,15 @@ class PreParforPass(object):
     implementations of numpy functions if available.
     """
     def __init__(self, func_ir, typemap, calltypes, typingctx, options,
-                 swapped={}):
+                 swapped={}, replace_functions_map=replace_functions_map):
```

Comment:
I'd suggest letting it defaulting to `None` and setting to the `replace_functions_map` global dictionary in the function body. With that, the test do not need to know what should be the default dict.